web/src/app/core/services/authentication.service.ts

```diff
@@ -15,6 +15,7 @@ import { concatMap, filter, first, switchMap, takeUntil, tap } from 'rxjs/operat
 import { DeviceDetectorService } from 'ngx-device-detector';
 
 // Dashboard hub models
+import * as firebase from 'firebase';
```

Comment:
This should probably go in the import group above

Reply:
Done